logutil/logutil.go

```diff
@@ -76,3 +76,42 @@ func NewServerLogger(debug bool) log.Logger {
 	}()
 	return &swapLogger
 }
+
+// NewFmtLogger creates a standard logger for Kolide CLI tools.
+// The logger will output leveled logs with a
+// "severity" field set to either "info" or "debug".
+// The acceptable level can be swapped by sending SIGUSR2 to the process.
+func NewFmtLogger(debug bool) log.Logger {
+	base := log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	base = log.With(base, "ts", log.DefaultTimestampUTC)
+	base = SetLevelKey(base, "severity")
```

Comment:
this is only necessary for processes running on GCP since the stackdriver logger can't infer severity from `level`. 
It also can't understand logfmt structured output, so it's not really necessary here.

Reply:
yeah, but it's nice to be consistent across all logs for consistent tooling support

Reply:
why not just change NewServerLogger` to take a format param? the functions are identical except for that

Reply:
I refactored the implementation into a shared helper, but I don't really want to change the public API of `NewServerLogger` and I like the idea of having consistent formats, so I left `NewCLILogger` and made `newLogger(debug bool, base log.Logger)` private.